mu-plugins/central-hub/tests/phpunit/integration/meta-data/getMetaBoxKeys.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Tests for the function get_meta_box_keys().
+ *
+ * @package     spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @since       1.3.0
+ * @author      Robert Gadon <rgadon107>
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Integration\Metadata;
+
+use function KnowTheCode\ConfigStore\loadConfig;
+use function spiralWebDB\Metadata\get_meta_box_keys;
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+
+/**
+ * Class Tests_GetMetaBoxKeys
+ *
+ * @package spiralWebDb\centralHub\Tests\Integration\Metadata
+ * @group   meta-data
+ */
+class Tests_GetMetaBoxKeys extends Test_Case {
+
+	/**
+	 * Empty the store before starting these tests.
+	 */
+	public static function setUpBeforeClass() {
+		self::empty_the_store();
+	}
+
+	/**
+	 * Clean up the test environment after each test.
+	 */
+	public function tearDown() {
+		self::empty_the_store();
+	}
+
+	/**
+	 * Test get_meta_box_keys() returns empty array when store key is empty.
+	 */
+	public function test_returns_empty_array_when_store_key_is_empty() {
+		loadConfig( '', [] );
+		get_meta_box_keys();
+
+		$this->assertArrayNotHasKey( 'meta_box.', get_meta_box_keys() );
+		$this->assertSame( [], get_meta_box_keys() );
+	}
+
+	/**
+	 * Test get_meta_box_keys() returns empty array when store key does not start with `meta_box`.
+	 */
+	public function test_returns_empty_array_when_store_key_does_not_start_with_meta_box() {
+		$configs = [
+			'notametabox.members' => [
+				'add_meta_box' => [
+					'id'      => 'members',
+					'screen'  => 'members',
+					'context' => 'normal',
+				],
+			],
+		];
+		loadConfig( 'notametabox.members', $configs['notametabox.members'] );
+		get_meta_box_keys();
```

Comment:
@rgadon107 There's no need to call `get_meta_box_keys()` on its own, unless you're running an assertion, as you have below.

Reply:
Removed.